src/PhpRenderer.php

```diff
@@ -82,6 +130,12 @@ public function fetch($template, array $data = []) {
             throw new \RuntimeException("View cannot render `$template` because the template does not exist");
         }
 
+        foreach ($data as $k=>$val) {
+            if (in_array($k, array_keys($this->attributes))) {
```

Comment:
I think that the one passed in `$data` should override the one in `$this->attributes`